site/pages/institution-contact-fr.hbs

```diff
@@ -0,0 +1,49 @@
+---
+{
+	"title": "Coordonnées [nom de l’institution]",
+	"language": "fr",
+	"altLangPrefix": "institution-contact",
+	"pageType": "institution",
+	"breadcrumb": [
+		{ "title": "Accueil", "link": "https://www.canada.ca/fr.html" },
+		{ "title": "Ministères et organismes", "link": "https://www.canada.ca/fr/gouvernement/min.html" }
+	],
+	"dateModified": "2017-09-29",
+	"share": "true"
+}
+---
+<div class="row">
+<div class="col-sm-7 pull-left">
```

Comment:
Can you do that block without the "pull-". Try to use instead "col-sm-pull-" and "col-sm-push-"